packages/forms/resources/views/components/text-input.blade.php

```diff
@@ -42,10 +42,11 @@
                 @else
                     x-data="textInputFormComponent({
                         {{ $hasMask() ? "getMaskOptionsUsing: (IMask) => ({$getJsonMaskConfiguration()})," : null }}
-                        state: $wire.{{ $applyStateBindingModifiers('entangle(\'' . $getStatePath() . '\')') }},
+                        state: $wire.{{ $applyStateBindingModifiers('entangle(\'' . $getStatePath() . '\')', except: 'lazy') }},
                     })"
                     type="text"
                     wire:ignore
+                    @if (in_array('lazy', $getStateBindingModifiers())) x-on:blur="$wire.$refresh" @endif
```

Comment:
We could maybe extract this into a `hasStateBindingModifier('lazy')` method?

Reply:
I it is going to be a separate method, I like short names ;) `@if(isLazy()) ... @endif`